src/opnsense/mvc/app/views/OPNsense/Unbound/overview.volt

```diff
@@ -483,6 +488,9 @@
                     if (window.localStorage.getItem("api.unbound.overview.logcchart") !== null) {
                         $("#toggle-log-cchart").prop('checked', window.localStorage.getItem("api.unbound.overview.logcchart") == 'true');
                     }
+                    if (window.localStorage.getItem("api.unbound.overview.extendeddomains") !== null) {
+                        $("#toggle-extended-domains").val(window.localStorage.getItem("api.unbound.overview.extendeddomains"));
+                    }
```

Comment:
To make this work, a `$('#toggle-extended-domains').selectpicker('refresh')` is required (see code below)